pkgs/development/libraries/science/astronomy/indilib/default.nix

```diff
@@ -45,14 +46,24 @@ stdenv.mkDerivation rec {
   cmakeFlags = [
     "-DCMAKE_INSTALL_LIBDIR=lib"
     "-DUDEVRULES_INSTALL_DIR=lib/udev/rules.d"
+  ] ++ lib.optional doCheck [
+    "-DINDI_BUILD_UNITTESTS=ON"
+    "-DINDI_BUILD_INTEGTESTS=ON"
   ];
 
+  checkInputs = [ gtest ];
+
+  doCheck = true;
```

Comment:
Do we want to disable this when doing cross?

Reply:
Isn't this the default behavior already?

https://github.com/NixOS/nixpkgs/blob/d75ba3d7a2b5c4952f89f498e435cb5648c0f0cb/pkgs/stdenv/generic/make-derivation.nix#L171